app/views/projects/show/_scan_analytics_summary.html.haml

```diff
@@ -1,17 +1,24 @@
 :ruby
-  version = @scan_data.analysis_metrics.version.present? ? @scan_data.analysis_metrics.version : nil
-  last_analysed = @scan_data.analysis_metrics.build_date
-  loc = @scan_data.analysis_metrics.loc
-  defect_density = @scan_data.analysis_metrics.defect_density&.score
-  total_defects = @scan_data.analysis_metrics.total_count
-  outstanding_count = @scan_data.analysis_metrics.outstanding_count
-  fixed_count = @scan_data.analysis_metrics.fixed_count
-.row.row-eq-height.project_row
-  %h4#scan_defect.text-left Static Analysis ( Generated by Coverity Scan with the url to <a href="#{@scan_data.project_url}" target='_blank'>Scan Project</a> )
-.row.project_row
-  .col-md-12#scan_defect
-    %h4.title Version: #{version}
-    .col-md-12
+  scan_data = JSON.parse(scan_analytics.data, object_class: OpenStruct)
+  version = scan_data.analysis_metrics.version.presence
+  last_analysed = scan_data.analysis_metrics.build_date
+  loc = scan_data.analysis_metrics.loc
+  defect_density = scan_data.analysis_metrics.defect_density&.score
+  total_defects = scan_data.analysis_metrics.total_count
+  outstanding_count = scan_data.analysis_metrics.outstanding_count
+  fixed_count = scan_data.analysis_metrics.fixed_count
```

Comment:
Reduce these ruby assignments like we do in this file:
https://github.com/blackducksoftware/ohloh-ui/blob/main/app/views/scan_analytics/_index.html.haml